### ISSUE TITLE
"velociraptor <distro> server" creates package for incorrect platform (amd64)

### ISSUE THREAD

**Issue by tstromberg** [Velocidex/velociraptor#2793]
I'm on an arm64 build server, and was able to build a working arm64 binary using `make`. 

However, when I use velociraptor to create a Debian server package, it creates an amd64 package, rather than matching the package of the underlying binary.

```
./output/velociraptor  --config server.config.yaml debian server --binary ./output/velociraptor 
Creating a package for velociraptor_0.7.0-dev_server.deb
```

Looks good, until:

```
$ sudo dpkg -i velociraptor_0.7.0-dev_server.deb                                                  
dpkg: error processing archive velociraptor_0.7.0-dev_server.deb (--install):
 package architecture (amd64) does not match system (arm64)
Errors were encountered while processing:
 velociraptor_0.7.0-dev_server.deb
```

**Comment by scudette** [comment]
Thanks for filing this issue. This was discussed previously here https://discord.com/channels/624244632552734750/624244632552734752/1115473108346884167 but we seemed to forget to file an issue to track it

**Comment by tstromberg** [comment]
I'll take a stab at sending a PR out today to address this.

**Comment by tstromberg** [comment]
Renaming since this issue impacts both `rpm` and `deb` packages. PR's incoming.

**Comment by scudette** [comment]
Should be fixed by #2797 and #2796
